test/src/test/kotlin/graphql/nadel/tests/EngineTests.kt

```diff
@@ -180,17 +192,20 @@ private suspend fun execute(
 
                                 if (indexOfCall != null) {
                                     val serviceCall = serviceCalls.removeAt(indexOfCall)
-                                    serviceCall.response
+                                    val response = serviceCall.response
```

Comment:
setting response to a variable, because kotlin thinks the state can change due to the way it was defined:
```
val response: JsonMap? by lazy {
...
}
```